src/main/main.c

```diff
@@ -155,11 +155,7 @@ static struct re global_re = {
 
 static void poll_close(struct re *re);
 
-#ifdef SOLARIS
-static pthread_once_t pt_once = {PTHREAD_ONCE_INIT};
-#else
 static pthread_once_t pt_once = PTHREAD_ONCE_INIT;
-#endif
 static pthread_key_t  pt_key;
```

Comment:
It makes the Solaris 11 build fail.  I did not try to build it on Illumos, but it seems fine without it.  It might have been needed on some really ancient version of Solaris, who knows...